client/js/app.js

```diff
@@ -59,17 +59,31 @@ const init = () => {
     makeSticky(stickyEls, store$);
   }
 
-  if (seriesNav) {
-    shrinkStoriesNav(seriesNav, dispatch);
-  }
-
-  if (seriesSlider) {
-    contentSlider(seriesSlider, {
-      transitionSpeed: 0.7,
-      startPosition: 0,
-      cssPrefix: 'numbered-list__'
+  const asyncSeriesNavAdded$ = store$.map((state) => {
+    return state.asyncComponents.some((item) => {
```

Comment:
This will be `null` if it doesn't exist - so you might want a filter:
```
const asyncSeriesNavAdded$ =
  store$.map((state) => state.asyncComponents.some(i => i === 'sticky-nav')).filter(i => i);
```
I'm also not sure why you use the `take(1)`?

Would this make sense in the asynContent component? It feels a bit without-a-home here?